### PR TITLE
Bug 1876562: Small readability improvements to UPI on RHV documentation

### PR DESCRIPTION
**Pull request by jan-zmeskal**
This just fixes couple of typos in UPI documentation for oVirt and attempts to make it a little bit more readable (though it was very good to begin with).

This also unifies `inventory.yaml` and `inventory.yml` into the latter form.

### CODE REVIEW

File: docs/user/ovirt/install_upi.md [hunk 1]
```diff
@@ -378,22 +385,22 @@ $ openshift-install create ignition-configs --dir ./wrk
 Other than the ignition files the installer generated
 
 - `auth` folder containing the admin credentials necessary to
-  connect to the cluster via the `oc` or `kubectl` CLIs.
-- `metadata.json` with informations like the ocp cluster name, ocp cluster ID and the `infraID`
-  (generated for the current running installation)
+  connect to the cluster via the `oc` or `kubectl` CLI utilities.
+- `metadata.json` with information like the OCP cluster name, OSC cluster ID and the `infraID`
```
Comment (gekorob) [inline docs/user/ovirt/install_upi.md:389]: Just a little typo in the correction of my many typos:
- can you change *OSC* into *OCP*

**Review by gekorob — APPROVED** [review]
Thanks a lot for your reviewing work and all the typos corrected.
This improves a lot the quality of the documentation

File: docs/user/ovirt/install_upi.md (hunk 1 above)
Comment (jan-zmeskal) [reply docs/user/ovirt/install_upi.md:389]: Sure :-D

Comment (jan-zmeskal) [reply docs/user/ovirt/install_upi.md]: Done

**Pull request merged**